draft-ietf-quic-transport.md

```diff
@@ -1619,10 +1627,10 @@ the legitimate copy will be dropped as a duplicate.
 
 After a spurious migration, validation of the source address will fail because
 the entity at the source address does not have the necessary cryptographic keys
-to read or respond to the PING frame that is sent to it, even if it wanted to.
-Such a spurious connection migration could result in the connection being
-dropped when the source address validation fails.  This grants an attacker the
-ability to terminate the connection.
+to read or respond to the PATH_CHALLENGE frame that is sent to it, even if it
+wanted to.  Such a spurious connection migration could result in the connection
+being dropped when the source address validation fails.  This grants an attacker
+the ability to terminate the connection.
```

Comment:
How about a recommendation to probe both old and new addresses to guard against this?

Reply:
I think Jana's planning to improve this in a follow up, so I'm ok with it as is.

Reply:
Yup. Happy to take this comment in the follow-up.